tst/regression/test_suites/advection_convergence/advection_convergence.py

```diff
@@ -56,11 +56,14 @@ def Prepare(self,parameters, step):
                 'mesh/nx1=400']
         """
 
+
         n_res = len(lin_res)
 
         # TEST: Advection only in x-direction 
         # using nx2 = nx3 = 4 > 1 for identical errors between dimensions
         if step <= n_res:
+
+            parameters.coverage_status = "both"
```

Comment:
I think it's fine to just run the coverage for the one case as for the y-direction.
```suggestion
            if lin_res[step - 1] == 32:
                parameters.coverage_status = "both"
```